elasticdl/pkg/commonnew/embedding_tabel.go

```diff
@@ -0,0 +1,70 @@
+package commonnew
```

Comment:
The file name `embedding_tabel.go` --> `embedding_table.go`

Reply:
fixed